src/main/java/core/basesyntax/model/FruitTransaction.java

```diff
@@ -0,0 +1,78 @@
+package core.basesyntax.model;
+
+public class FruitTransaction {
+    private Operation operation;
+    private String fruit;
+    private int quantity;
+
+    public FruitTransaction(Operation operation, String fruit, int quantity) {
+        this.operation = operation;
+        this.fruit = fruit;
+        this.quantity = quantity;
+    }
+
+    public FruitTransaction() {
+    }
+
+    public Operation getOperation() {
+        return operation;
+    }
+
+    public void setOperation(String code) {
+        this.operation = Operation.fromCode(code);
+        //return this;
+    }
+
+    public String getFruit() {
+        return fruit;
+    }
+
+    public void setFruit(String fruit) {
+        this.fruit = fruit;
+        //return this;
+    }
+
+    public int getQuantity() {
+        return quantity;
+    }
+
+    public void setQuantity(int quantity) {
+        this.quantity = quantity;
+        //return this;

```

Comment:
Commented code found. It's a bad practice to leave commented code in production codebase.

Reply:
ok, done